PlaylistManager/UI/ViewControllers/PlaylistDetailViewButtonsController.cs

```diff
@@ -94,7 +94,7 @@ private void ShowDetails()
         [UIAction("delete-click")]
         private void OnDelete()
         {
-            var numberOfSongs = selectedPlaylist.PlaylistLevelPack.beatmapLevels.Length;
+            var numberOfSongs = selectedPlaylist.PlaylistLevelPack._beatmapLevels.Length;
```

Comment:
Again this should be `.AllBeatmapLevels()`.

Reply:
Fixed, although is there even a difference between just using the method or the field directly instead?

Reply:
The difference is that if they decide to change the logic of the method then you'll have no change to make.